web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -246,7 +247,7 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
                 name: jobName,
                 revisions: revisionId ? [revisionId] : revisions,
                 datasets,
-                files,
+                files: (files?.length ?? 0) > 0 ? files : fileId ? [fileId] : [],
```

Comment:
what about this? will it work?
```suggestion
                files: files?.length ? files : fileId ? [fileId] : [],
```